src/Symfony/Bridge/PhpUnit/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ CHANGELOG
  * ignore verbosity settings when the build fails because of deprecations
  * added per-group verbosity
  * added `ExpectDeprecationTrait` to be able to define an expected deprecation from inside a test
+ * deprecated the `@expectedDeprecation` annotation, use the `ExpectDeprecationTrait::expectDeprecation()` method instead
```

Comment:
we need a similar message in UPGRADE-5.1 also at the root

Reply:
@nicolas-grekas Done.